src/index.js

```diff
@@ -1,2 +1,32 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable no-unused-vars */
+/* eslint-disable import/no-mutable-exports */
+/* eslint-disable no-plusplus */
+// eslint-disable-next-line no-unused-vars
```

Comment:
I think it would be a very good idea to avoid disabling linter configurations and fixing linter errors as they arise. This is because these linter checks are made to ensure guidelines and best practices that are accepted in microverse and in the programming community and disabling these checks means not following these guidelines 😉